Lib/_pydecimal.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2004 Python Software Foundation.
+# Copyright (c) 2017 Python Software Foundation.
```

Comment:
Perhaps on these where the source was a single year, it should convert to `2004-2017`